pkg/symbolizer/symbolizer_test.go

```diff
@@ -25,6 +25,7 @@ func TestParse(t *testing.T) {
 				"mm/kasan/report.c:320\n",
 			[]Frame{
 				{
+					Module: "kernel",
```

Comment:
I think it would be better convention to use an empty string to denote the main kernel. Empty module name means "not a module", which means main kernel.
It would avoid adding lines like this one, and lots of duplication of "kernel" string.